tests/main/snapd-snap/task.yaml

```diff
@@ -12,5 +12,5 @@ restore: |
 execute: |
     echo "Installing snapscraft"
     apt install -y snapcraft
-    cd $PROJECT_PATH
+    cd "$PROJECT_PATH" || exit
```

Comment:
This is not really needed because we run our spread with `set -e` so cd failing will result in an abort. But it seems like shellcheck does not know about us running with set -e :/ I wonder if we can generate it automatically in the inputs we feed to shellcheck?